.circleci/config.yml

```diff
@@ -45,21 +45,6 @@ jobs:
           paths:
             - /cache/docker.tar
 
-  test:
-    docker:
-      - image: docker:18.02.0-ce
-    steps:
-      - setup_remote_docker
-      - restore_cache:
-          key: v1-{{ .Branch }}-{{ .Environment.CIRCLE_TAG }}
-      - run:
-          name: Restore Docker image cache
-          command: docker load -i /cache/docker.tar
-
-      - run:
-          name: Test Code
-          command: docker run app:build test
```

Comment:
Seems like this `test` target can't possible work given the way the docker image is built, so I removed it.